plugins/custom-cols.yaml

```diff
@@ -29,8 +29,9 @@ spec:
     It's time saving to retrieve some often needed infos like
     image names, containers names, versions, labels, etc...
     It currently supports the following templates (but PRs are welcome upstream):
```

Comment:
it might be wise to not include this sentence and what's below so you can get these PRs auto-approved. you can also use release automation not to craft these PRs yourself https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/

Reply:
ok thanks for the merge any may @ahmetb i will definitely look into it